UseCases/Package.swift

```diff
@@ -10,8 +10,7 @@ let package = Package(
     ],
     dependencies: [
         .package(name: "gsoc-swift-tracing", path: "../"),
-        .package(url: "https://github.com/swift-server/async-http-client.git", from: "1.1.1"),
-        .package(url: "https://github.com/apple/swift-nio.git", from: "2.18.0")
+        .package(url: "https://github.com/swift-server/async-http-client.git", from: "1.1.1")
```

Comment:
1) would it work if you depended on httpclient 1.6.0 and nio 2.9?

Reply:
2. nitpick, please keep the trailing `,` in `[]` listings like here

Reply:
There's no 1.6.0 of `AsyncHTTPClient` 🤔 Latest is 1.1.1: https://github.com/swift-server/async-http-client/releases

Reply:
Using NIO 2.9 & HTTP Client 1.1.1 also fixes the issue. What solution would you prefer?

Reply:
Let's include specific versions NIO 2.9 & HTTP Client 1.1.1 is ok.